src/IO/S3/PocoHTTPClient.h

```diff
@@ -169,6 +171,8 @@ class PocoHTTPClient : public Aws::Http::HttpClient
         Aws::Utils::RateLimits::RateLimiterInterface * readLimiter,
         Aws::Utils::RateLimits::RateLimiterInterface * writeLimiter) const;
 
+    ConnectionTimeouts getTimeouts(const String & method, bool first_attempt, bool first_byte = true) const;
```

Comment:
minor, but imo would be better without default value for `first_byte`

Reply:
Ok, will remove default value here.